csp-mixin/panels.libsonnet

```diff
@@ -899,5 +899,551 @@ local commonlib = import 'common-lib/common/main.libsonnet';
       + g.panel.timeSeries.fieldConfig.defaults.custom.stacking.withMode('normal')
       + this.signals.azurevirtualnetwork.tcpPacketsForwarded.asPanelMixin()
       + this.signals.azurevirtualnetwork.udpPacketsForwarded.asPanelMixin(),
+
+    // Azure Virtual Machines
```

Comment:
It looks like all of panels are still defined in this file?

The panels only need to be defined in one place, so this file can just become a single import statement.

Thank you for this re-organization. I realized after the first couple dashboards that we should have done this from the start 😂

Reply:
removed!